ACPs/77-reinventing-subnets/README.md

```diff
@@ -103,22 +111,27 @@ After the `RegisterSubnetValidatorTx` is accepted on the P-Chain, the Subnet Val
 
 When any validator is removed from the set (whether forcefully or per the validator's request), the P-Chain will also send a warp message to the Subnet notifying it of the validator set removal. It is up to the Subnet on how to handle such a message, especially if unexpected. A validator's stake could continue to remain locked for an extended period of time after this point, for example.
 
-Note: There is no `EndTime` specified in this transaction. Subnet Validators are only evicted when an `EvictSubnetValidatorTx` or `ExitValidatorTx` is issued.
+This transaction is, by design, not required to be submitted by the validator themselves. Through the Ed25519 signature, the validator guarantees that they can only be added to the validator set if and only if `Signer` corresponds to the `blsPublicKey` and `Balance` >= `balance`. The `RegisterSubnetValidatorTx` is considered invalid if those two properties are not satisfied.
+
+Note: There is no `EndTime` specified in this transaction. Subnet Validators are only evicted when a `SetSubnetValidatorWeightTx` sets a validator's weight to `0` or `ExitValidatorTx` is issued.
 
-#### EvictSubnetValidatorTx
+#### SetSubnetValidatorWeightTx
 
```

Comment:
I would add a note that this can allow a Subnet Validator to stake rewards without re-becoming a validator?

Reply:
(not sure why this comment showed on this side)